BIO-A or OCC-AUTH before in person PIN reset (#638)
usnistgov/FIPS201: _FIPS201/requirements.md

remotely via a general computing platform or a supervised remote identity proofing station:

In person

> : When PIN reset is performed in person at the issuing facility, before
>     resetting the PIV Card's PIN, the issuer **SHALL** first perform a biometric verification to ensure that the cardholder's
>     biometric characteristics elicit a positive biometric verification decision when compared to biometric data records stored in the PIV enrollment record or when compared to the biometric data records on the PIV Card using the BIO-A or OCC-AUTH authentication mechanisms. In cases where a negative biometric verification decision is returned or the cardholder's biometric characteristics are not successfully acquired, the cardholder **SHALL** provide the PIV Card

Reviewer: ```suggestion
: When PIN reset is performed in person at the issuing facility, before
    resetting the PIV Card's PIN, the issuer **SHALL** first perform a biometric verification to ensure that the cardholder's
    biometric characteristics elicit a positive biometric verification decision when compared to biometric data records stored in the PIV enrollment record or when compared to the biometric data records on the PIV Card using OCC-AUTH authentication mechanisms. Alternatively, BIO-A authentication mechanism **MAY** be used as biometric verification after the card has been reset and while the card is in the operator's custody. The operator then hands the reset PIV Card to the cardholder. In cases where a negative biometric verification decision is returned or the cardholder's biometric characteristics are not successfully acquired, the cardholder **SHALL** provide the PIV Card
```
```suggestion
: When PIN reset is performed in person at the issuing facility, before
    resetting the PIV Card's PIN, the issuer **SHALL** first perform a biometric verification to ensure that the cardholder's
    biometric characteristics elicit a positive biometric verification decision when compared to biometric data records stored in the PIV enrollment record or when compared to the biometric data records on the PIV Card using the BIO-A or OCC-AUTH authentication mechanisms. In cases where a negative biometric verification decision is returned or the cardholder's biometric characteristics are not successfully acquired, the cardholder **SHALL** provide the PIV Card
```
```suggestion
: When PIN reset is performed in person at the issuing facility, before
    resetting the PIV Card's PIN, the issuer **SHALL** first perform a biometric verification to ensure that the cardholder's
    biometric characteristics elicit a positive biometric verification decision when compared to biometric data records stored in the PIV enrollment record or when compared to the biometric data records on the PIV Card using the BIO-A or OCC-AUTH authentication mechanisms. In cases where a negative biometric verification decision is returned or the cardholder's biometric characteristics are not successfully acquired, the cardholder **SHALL** provide the PIV Card
```
```suggestion
: When PIN reset is performed in person at the issuing facility, before
    resetting the PIV Card's PIN, the issuer **SHALL** first perform a biometric verification to ensure that the cardholder's
    biometric characteristics elicit a positive biometric verification decision when compared to biometric data records stored in the PIV enrollment record or when compared to the biometric data records on the PIV Card using the BIO-A or OCC-AUTH authentication mechanisms. In cases where a negative biometric verification decision is returned or the cardholder's biometric characteristics are not successfully acquired, the cardholder **SHALL** provide the PIV Card
```

remotely via a general computing platform or a supervised remote identity proofing station:

In person
: When PIN reset is performed in person at the issuing facility, before

>     resetting the PIV Card's PIN, the issuer **SHALL** first perform a biometric verification to ensure that the cardholder's

Author: ```suggestion
    resetting the PIV Card's PIN, the issuer **SHALL** perform a biometric verification to ensure that the cardholder's
```

>     biometric characteristics elicit a positive biometric verification decision when compared to biometric data records stored in the PIV enrollment record or when compared to the biometric data records on the PIV Card using the OCC-AUTH authentication mechanism. In cases where a negative biometric verification decision is returned or the cardholder's biometric characteristics are not successfully acquired, the cardholder **SHALL** provide the PIV Card

Author: ```suggestion
    biometric characteristics elicit a positive biometric verification decision when compared to biometric data records stored in the PIV enrollment record or when compared to the biometric data records on the PIV Card using the BIO-A or OCC-AUTH authentication mechanisms. In cases where a negative biometric verification decision is returned or the cardholder's biometric characteristics are not successfully acquired, the cardholder **SHALL** provide the PIV Card
```
